app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -3,6 +3,11 @@
   helpdesk_email = Rails.configuration.x.organisation.helpdesk_email
   contact_us = Rails.configuration.x.organisation.contact_us_url || contact_us_url
   email_subject = _('Query or feedback related to %{tool_name}') %{ :tool_name => tool_name }
+  user = User.find_by(email: @resource.email)
+  helpdesk_email = user.org.present? &&

```

Comment:
Should be able to use the helper method here

Reply:
@briri  How do I include MailerHelper in Devise views?

I tried but got error "undefined method `include' for #<#<Class:0x00007fb1cd01b310>:0x00007fb1c0e9e750>"

 ```
 <%
  include MailerHelper
  helper MailerHelper

  tool_name = ApplicationService.application_name
  helpdesk_email = Rails.configuration.x.organisation.helpdesk_email
  contact_us = Rails.configuration.x.organisation.contact_us_url || contact_us_url
  email_subject = _('Query or feedback related to %{tool_name}') %{ :tool_name => tool_name }
  user = User.find_by(email: @resource.email)
  helpdesk_email =  helpdesk_email(user.org)
%>
```